implicits/src-3/upickle/implicits/CaseClassReader.scala

```diff
@@ -1,8 +1,12 @@
 package upickle.implicits
 
-import compiletime.{summonInline}
+import compiletime.summonInline
 import deriving.Mirror
-import upickle.core.{ Visitor, ObjVisitor, Annotator }
+import upickle.core.{Annotator, ObjVisitor, Visitor, Abort}
+import upickle.implicits.macros.EnumDescription
+
+import deriving.*
+import compiletime.*
```

Comment:
unless members of these are truly used everywhere in this file and it is clear from context, please avoid using wildcard imports